lua/mason-registry/misspell/init.lua

```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local go = require "mason-core.managers.go"
+
+return Pkg.new {
+    name = "misspell",
+    desc = [[Correct commonly misspelled English words in source files]],
+    homepage = "https://github.com/client9/misspell",
+    languages = { Pkg.Lang.Go },
```

Comment:
```suggestion
    languages = {},
```

(it would be nice to have a metalanguage specifying "all" languages)